manifests/l/Lazarus/Lazarus/3.8/Lazarus.Lazarus.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=QUSU.CRLF.7-4-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Lazarus.Lazarus
+PackageVersion: "3.8"
+InstallerLocale: en-US
+InstallerType: inno
+AppsAndFeaturesEntries:
+- Publisher: Lazarus Team
+  ProductCode: '{code:GetAppId}_is1'
```

Comment:
This doesn't look right

Reply:
I can't inspect it at this moment. The provided `ProductCode` was inspected by _Komac_.

Shall I remove it here directly?